src/solver/variable/economy/all.h

```diff
@@ -125,60 +124,57 @@ class Links;
-                              <LOLP                // LOLP
-                               <AvailableDispatchGen
-                                <DispatchableGenMargin
-                                <DtgMarginCsr // DTG MRG CSR
-                                 <Marge<NonProportionalCost<
-                                   NonProportionalCostByDispatchablePlant // Startup cost + Fixed
-                                                                          // cost per thermal
-                                                                          // plant detail
-                                   <NbOfDispatchedUnits         // Number of Units Dispatched
-                                    <NbOfDispatchedUnitsByPlant // Number of Units Dispatched by
-                                                                // plant
-                                     <ProfitByPlant
-                                      // Links
-                                      <Variable::Economy::Links // All links
-                                       >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+typedef                           // Prices
+  OverallCost                     // Overall Cost (Op. Cost + Unsupplied Eng.)
+  <OperatingCost                  // Operating Cost
+   <Price                         // Marginal price
+                                  // Thermal pollutants
+    <ThermalAirPollutantEmissions // Overall pollutant emissions(from all thermal dispatchable
+                                  // clusters) Production by thermal cluster
+     <ProductionByDispatchablePlant    // Energy generated by thermal dispatchable clusters
+      <ProductionByRenewablePlant      // Energy generated by renewable clusters (must-run)
+       <Balance                        // Nodal Energy Balance
+                                       // Misc Gen.
+        <RowBalance                    // Misc Gen. Row balance
+         <PSP                          // PSP
+          <MiscGenMinusRowPSP          // Misc Gen. - Row Balance - PSP
+                                       // Time series
+           <TimeSeriesValuesLoad       // Load
+            <TimeSeriesValuesHydro     // Hydro
+             <TimeSeriesValuesWind     // Wind
+              <TimeSeriesValuesSolar   // Solar
+                                       // Other
+               <DispatchableGeneration // All dispatchable generation
+                <RenewableGeneration   // All renewable generation
+                 <HydroStorage         // Hydro Storage Generation
+                  <Pumping             // Pumping generation
+                   <ReservoirLevel     // Reservoir levels
+                    <Inflows           // Hydraulic inflows
+                     <Overflows        // Hydraulic overflows
+                      <WaterValue      // Water values
+                       <HydroCost      // Hydro costs
+                        <ShortTermStorageByGroup<STstorageInjectionByCluster<
```

Comment:
**Indentation** : 
why folding the ST storage variable here (**economy/all.h**) and not in **adequacy/all.h** ?
To me, unfolding them is more clear.

Reply:
I only used clang-format, nothing else.

Reply:
I could format by hand, but I don't really want to